plugins/bulk_upload/csv/batch/BulkUploadVendorCatalogItemEngineCsv.php

```diff
@@ -257,6 +247,7 @@ protected function createObjects()
 				case KalturaBulkUploadAction::UPDATE:
 					$bulkUploadResultChunk[] = $bulkUploadResult;
 					$vendorCatalogItem = $this->createVendorCatalogItemFromResult($bulkUploadResult);
+					$vendorCatalogItem->bulkUpdateId = $this->job->id;

```

Comment:
i think we should change it to lastBulkUpdateId on the second pr to make it more clear whats this field reflects